en/documentation/installation/index.md

```diff
@@ -217,6 +218,15 @@ $ pkg install runtime/ruby
 However, the third-party tools might be a good way to obtain the
 latest version of Ruby.
 
+### WinGet
```

Comment:
Should it be all lowercase per https://docs.microsoft.com/en-us/windows/package-manager/winget/?

<img width="959" alt="CleanShot 2021-09-07 at 21 53 34@2x" src="https://user-images.githubusercontent.com/1000669/132347965-afccc480-20e1-4b8c-99a0-345897482f1f.png">

Reply:
Looking at recent docs (https://devblogs.microsoft.com/commandline/windows-package-manager-1-0/) I think the naming is shifting towards **Windows Package Manager**, so maybe use this name?

The [repo](https://github.com/microsoft/winget-cli/) refers to `winget.exe` as **Windows Package Manager Client**.

Reply:
Windows Package Manager sounds good and clear to folks not yet aware of this tool like me 👍🏼  Very cool.

Reply:
The lowercase is cli name, not software. I picked from https://github.com/microsoft/winget-cli#manually-update.